openapi-contracts/src/main/com/sinch/sdk/domains/conversation/models/dto/v1/AgentLeftEventDto.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Conversation API | Sinch
+ * Send and receive messages globally over SMS, RCS, WhatsApp, Viber Business, Facebook messenger and other popular channels using the Sinch Conversation API.  The Conversation API endpoint uses built-in transcoding to give you the power of conversation across all supported channels and, if required, full control over channel specific features.
+ *
+ * The version of the OpenAPI document: 1.0
+ * Contact: support@sinch.com
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+package com.sinch.sdk.domains.conversation.models.dto.v1;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import java.util.Objects;
+
+/** AgentLeftEventDto */
+@JsonPropertyOrder({AgentLeftEventDto.JSON_PROPERTY_AGENT_LEFT_EVENT})
+@JsonFilter("uninitializedFilter")
+@JsonInclude(value = JsonInclude.Include.CUSTOM)
+public class AgentLeftEventDto {
```

Comment:
Same question as for the `AgentJoinedEventDto` class

Reply:
Same assumption (like for AgentJoinedEvent)